lesson07/assignment/src/views/accordion.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div class="accordion">
+       <section class="main_accordion">
+            <div id="accordion">
+                <div v-for="poster in posters" class="card">
+                    <div class="card-header">
+                        <a v-bind:href="'#' + poster.div_id" data-parent="#accordion"
+                            data-toggle="collapse">
+                            <img v-bind:src="poster.img_link" alt="">
+                        </a>
+                    </div>
+                    <div v-bind:id="poster.div_id" class="collapse">
+                        <div class="card-body">
+                            <h3 class="display-4" data-toggle="modal" v-bind:data-target="'.' + poster.div_class">{{poster.movie_name}}</h3>
+                            <div v-bind:class="'modal fade ' + poster.div_class" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel" aria-hidden="true">
+                                <div class="modal-dialog modal-lg">
+                                    <div class="modal-content">
+                                     {{poster.modal_line_1}}<br>
```

Comment:
Rather than using `<br>`, I'd wrap each content in a `<p>` and use margin to set the spacing